Makefile

```diff
@@ -3,6 +3,8 @@ IMAGE_LAMBDA_GO=eawsy/aws-lambda-go-shim
 GOPATH ?= $(HOME)/go
 HANDLER ?= handler
 PACKAGE ?= package
+AWS_ACCOUNT_NUMBER ?= $(shell aws iam list-users --output text | head -1|  cut -f2 | cut -f5 -d:)
```

Comment:
This adds an unnecessary dependency of depending on the aws-cli to be properly configured with a default profile. Why not just generate the Lambda zip file and use serverless to upload the artifact to the proper AWS account?